puzzles/templates/story.html

```diff
@@ -1,37 +1,38 @@
 {% extends "base.html" %}
+{% load i18n %}
 {% block content %}
 
-<h1>Story</h1>
+<h1>{% translate "Story" %}</h1>
 
 <main>
 
 {% for key in story_points %}
 
 {% if key == 'meta2_done' %}
 <div class="no-break story-chunk">
-    <h4>Meta Meta</h4>
-    You succeeded!
+    <h4>{% translate "Meta Meta" %}</h4>
+    {% translate "You succeeded!" %}
```

Comment:
I had this dilemma. I did this because it's a bit surprising to get some English text on a site that has been translated in another language and it doesn't cost much to have those. Maybe a FIXME could be added in there.